modules/linear_boltzmann_solvers/b_diffusion_dfem_solver/iterative_methods/mip_wgs_context2.cc

```diff
@@ -12,14 +12,13 @@
 
 #include <iomanip>
 
-#define sc_double static_cast<double>
-#define PCShellPtr PetscErrorCode (*)(PC, Vec, Vec)
-
 namespace opensn
 {
 namespace lbs
 {
 
+typedef PetscErrorCode (*PCShellPtr)(PC, Vec, Vec);
```

Comment:
We should discuss `using` vs `typedef` at some point

Reply:
I think we should be using `using`. This is just an old habit of mine from pre-C++11 days.